gbpservice/nfp/common/constants.py

```diff
@@ -42,6 +42,14 @@
 PENDING_DELETE = "PENDING_DELETE"
 ERROR = "ERROR"
 
+# Constants to extend status strings in neutron.plugins.common.constants
+ONLINE = 'ONLINE'
+OFFLINE = 'OFFLINE'
+DEGRADED = 'DEGRADED'
+DISABLED = 'DISABLED'
+NO_MONITOR = 'NO_MONITOR'
+OPERATING_STATUSES = (ONLINE, OFFLINE, DEGRADED, DISABLED, NO_MONITOR)
```

Comment:
Question:

For those ONLINE, OFFLINE, etc., I only use it in gbpservice/nfp/config_orchestrator/agent/loadbalancerv2.py. I don't think it is good to put them in nfp/common/constants. 

Do you guys have any suggestion where I should put them? Because there is no such a constant folder under config_orchestrator.